api/src/test/java/gov/cms/ab2d/api/controller/BulkDataAccessAPIIntegrationTests.java

```diff
@@ -1148,4 +1148,20 @@ private ResultMatcher buildExpiresMatcher() {
                     " should be greater than expected - " + minExpected);
         };
     }
+
+    /*
+     * Be more accepting of a one-second difference in timestamps when running a test.
+     */
+    private ResultMatcher buildTxTimeMatcher() {
+        return result -> {
+            OffsetDateTime buildTime = OffsetDateTime.now();
+            String buildTimeStr = new org.hl7.fhir.dstu3.model.DateTimeType(buildTime.toString()).toHumanDisplay();
```

Comment:
I want to say this is overly convoluted but I wouldn't be surprised if this is the best way to convert between these two datatime types.

Reply:
I left the original convulsion in place and just added tolerance for deviation by a second.  Matchers are a bit tricky to work with.  At lease having the duplicate code in one builder method is simpler and more contained.